llvm/lib/Target/AArch64/MCTargetDesc/AArch64AsmBackend.cpp

```diff
@@ -446,7 +446,7 @@ void AArch64AsmBackend::applyFixup(const MCAssembler &Asm, const MCFixup &Fixup,
   // Copy the Value at the fixup location and zero-out the fixup if it is
   // unresolved, this is done to improve deduplication with MCCAS. The fixup
   // will be applied later when the object file is being written out.
-  if (!IsResolved) {
+  if (!IsResolved && Fragment->getParent()->getName() != "__debug_info") {
```

Comment:
Since this is non-MachO-specific code, we need to do something more generic than check fort the Mach-O section name here.

Reply:
I thought the CAS was only supported for MachO, we do not want to zero out the addends for a MachO debug info section right?